cogs/invite.py

```diff
@@ -264,6 +269,11 @@ async def invite_close(self, ctx, code=None):
 
     @invite.command(name="list")
     async def invite_list(self, ctx):
+        """
+        List all open invites
+
+        These are only invites that were opened/requested by the user.
```

Comment:
```suggestion
        Unless the user is allowed to manage invites through the bot, this will just list invites that the user created or approved.
```